stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -417,13 +417,27 @@ internal struct _ContiguousArrayBuffer<Element>: _ArrayBufferProtocol {
   }
 
 #if _runtime(_ObjC)
+  
   /// Convert to an NSArray.
   ///
   /// - Precondition: `Element` is bridged to Objective-C.
   ///
   /// - Complexity: O(1).
-  @inlinable
+  @usableFromInline
```

Comment:
I know these seems pedantic, but could you add a comment after the `@usableFromInline`? Something to the effect of:

```swift
@usableFromInline // was @inlinable through Swift 5.1 (see comment below)
```

I want some indication right on the decl of the binary compatibility story, without reading comments inside the body (which may be about the implementation).

Reply:
Can we also mark these deprecated at this point?

Reply:
Can we mark things as deprecated without breaking our own/Foundation's callsites?